src/Microsoft.NET.Sdk.Aspire/Microsoft.NET.Sdk.Aspire.csproj

```diff
@@ -18,11 +18,7 @@
 
   <ItemGroup>
     <TextReplacementValue Include="MicrosoftDotnetSdkInternalPackageVersion" NewValue="$(MicrosoftDotnetSdkInternalPackageVersion)" />
-    <TextReplacementValue Include="MicrosoftNETCoreAppRefPackageVersion" NewValue="$(MicrosoftNETCoreAppRefPackageVersion)" />
```

Comment:
FYI @joperezr - I searched the repo for these strings and couldn't find them being used anymore. So I removed them. If you know that they are still used, please let me know.

Reply:
Yeah, they were used by our project that builds the manifest and does the replacements, but we have changed the template manifest and targets, so if it's not used in any of the *.in.* files here: https://github.com/dotnet/aspire/tree/main/src/Microsoft.NET.Sdk.Aspire